libs/vkd3d/vkd3d_private.h

```diff
@@ -4885,6 +4932,36 @@ UINT d3d12_determine_shading_rate_image_tile_size(struct d3d12_device *device);
 bool d3d12_device_supports_required_subgroup_size_for_stage(
         struct d3d12_device *device, VkShaderStageFlagBits stage);
 
+static inline void d3d12_device_register_swapchain(struct d3d12_device *device, struct dxgi_vk_swap_chain *chain)
+{
+    spinlock_acquire(&device->low_latency_swapchain_spinlock);
+
+    dxgi_vk_swap_chain_incref(chain);
```

Comment:
incref is only required when assigning to low_latency_swapchain member. That way it's more obvious why the incref is done.

Reply:
Done.